db/migrate/20170505141654_create_departments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateDepartments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :departments do |t|
+      t.string :name
```

Comment:
would be a kind of tricky thing. no? How do I validate presence in departement, but not in clients or profiles later on?